### ISSUE TITLE
Should we change the eARG notation to remove natural numbers

### ISSUE THREAD

**Issue by hyanwong** [tskit-dev/what-is-an-arg-paper#205]
I didd bring this up before, but now that we are labelling the nodes in the new Fig 1 with letters, should we change the definition of the mapping from e.g. `$\sigma: \mathbb{N} \rightarrow \mathbb{N}$` to something that doesn't imply that the node id is a natural number, but is simply a member of a set of identifiers?

![Screenshot 2022-04-07 at 10 45 51](https://user-images.githubusercontent.com/4699014/162171417-77adaaaa-5d33-4f83-874a-6c5ff15d953d.png)

**Comment by jeromekelleher** [comment]
No, let's keep this concrete. People can understand that A etc refer to natural numbers. There's nothing to be gained from making the definition more general and it will distract from the point that we *want* people to think of nodes concretely as numbers.

**Comment by hyanwong** [comment]
I'm finessing this section and will more the reference to the figure to the previous para, so the figure labelling and the definition is not quite such a jarring contrast anyway.
